packages/@ourworldindata/grapher/src/slopeCharts/SlopeChart.tsx

```diff
@@ -836,484 +427,636 @@ class LabelledSlopes
-                    } else {
-                        this.props.onMouseLeave()
-                    }
-                }
-            })
-        }
+        this.onSlopeMouseLeave()
     }
 
-    @action.bound onClick() {
-        if (this.props.onClick) this.props.onClick()
+    @computed get failMessage(): string {
+        const message = getDefaultFailMessage(this.manager)
+        if (message) return message
+        else if (this.startTime === this.endTime)
+            return "No data to display for the selected time period"
+        return ""
     }
 
-    componentDidMount() {
-        if (!this.manager.disableIntroAnimation) {
-            this.playIntroAnimation()
-        }
+    @computed get helpMessage(): string | undefined {
+        if (
+            this.failMessage ===
+            "No data to display for the selected time period"
+        )
+            return "Try dragging the time slider to display data."
+        return undefined
     }
 
-    private playIntroAnimation() {
-        // Nice little intro animation
-        select(this.base.current)
-            .select(".slopes")
-            .attr("stroke-dasharray", "100%")
-            .attr("stroke-dashoffset", "100%")
-            .transition()
-            .attr("stroke-dashoffset", "0%")
+    @computed get renderUid(): number {
+        return guid()
     }
 
-    renderGroups(groups: SlopeEntryProps[]) {
-        const { isLayerMode, isMultiHoverMode } = this
+    @computed get tooltip(): React.ReactElement | undefined {
+        const {
+            manager: { isRelativeMode },
+            tooltipState: { target, position, fading },
+            formatColumn,
+            startTime,
+            endTime,
+        } = this
+
+        const { series } = target || {}
+        if (!series) return
+
+        const title = isRelativeMode
+            ? `${series.seriesName}, ${formatColumn.formatTime(endTime)}`
+            : series.seriesName
+
+        const timeRange = [startTime, endTime]
+            .map((t) => formatColumn.formatTime(t))
+            .join(" to ")
+        const timeLabel = isRelativeMode
+            ? `% change since ${formatColumn.formatTime(startTime)}`
+            : timeRange
+
+        const columns = this.yColumns
+        const allRoundedToSigFigs = columns.every(
+            (column) => column.roundsToSignificantFigures
+        )
+        const anyRoundedToSigFigs = columns.some(
+            (column) => column.roundsToSignificantFigures
+        )
+        const sigFigs = excludeUndefined(
+            columns.map((column) =>
+                column.roundsToSignificantFigures
+                    ? column.numSignificantFigures
+                    : undefined
+            )
+        )
+
+        const roundingNotice = anyRoundedToSigFigs
+            ? {
+                  icon: allRoundedToSigFigs
+                      ? TooltipFooterIcon.none
+                      : TooltipFooterIcon.significance,
```

Comment:
If some values are rounded to sig figs and other are not, then we put an icon next to each value that is rounded to sig figs – and then explain the icon in the footer. But if all values are rounded to sig figs, we don't need the icons. The footer simply says "All values are rounded to sig figs".

That being said, it doesn't make sense here to take into account all y-columns since we only show the value of one of those columns. Fixing it!